src/main/java/seedu/address/logic/commands/SearchCommand.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Format search instructions for every command for display.
+ */
+public class SearchCommand extends Command {
```

Comment:
Would it be better to change the class name to SearchTagCommand? Since there is a possibility of implementing search by person in the future

Reply:
yes. I have renamed it to search tag. As of now, for the current search command, it is named as search only as no implementation of searching by other means exist yet.